### PR TITLE
<DEVOPS-134> Creation of topic with pull subscription

### PR DESCRIPTION
**Pull request by pankajholariyacb**

### CODE REVIEW

File: payment_pubsub.tf [hunk 1]
```diff
@@ -26,3 +26,16 @@ module "payment_ticket_events_push" {
     }
   ]
 }
+
+module "payment-refunds" {
+  source  = "terraform-google-modules/pubsub/google"
+  version = "~> 1.8"
```
Comment (Sunil-Jacob) [inline payment_pubsub.tf:32]: @pankajholariyacb Remove this line `version = "~> 1.8"`

**Review by Sunil-Jacob — COMMENTED** [review]
Requesting changes

File: payment_pubsub.tf (hunk 1 above)
Comment (pankajholariyacb) [reply payment_pubsub.tf]: Hi Sir, changes has been done.

**Review by Sunil-Jacob — APPROVED** [review]
👍

**Pull request merged**